specification/migrate/resource-manager/Microsoft.Migrate/2017-11-11-preview/migrate.json

```diff
@@ -0,0 +1,2763 @@
+            "modelAsString": true
+          }
+        },
+        "datacenterContainer": {
+          "type": "string",
+          "description": "Container defined in the management solution that this machine is part of in the datacenter.",
+          "readOnly": true
+        },
+        "datacenterManagementServer": {
+          "type": "string",
+          "description": "Name of the server hosting the datacenter management solution.",
+          "readOnly": true
+        },
+        "datacenterMachineId": {
+          "type": "string",
+          "description": "ID of the machine as tracked by the datacenter management solution.",
+          "readOnly": true
+        },
+        "datacenterManagementServerId": {
+          "type": "string",
+          "description": "ID of the server hosting the datacenter management solution.",
+          "readOnly": true
+        },
+        "description": {
+          "type": "string",
+          "description": "Description of the machine",
+          "readOnly": true
+        },
+        "displayName": {
+          "type": "string",
+          "description": "User readable name of the machine as defined by the user in their private datacenter.",
+          "readOnly": true
+        },
+        "megabytesOfMemory": {
+          "type": "number",
+          "description": "Memory in Megabytes.",
+          "readOnly": true
+        },
+        "numberOfCores": {
+          "type": "integer",
+          "description": "Processor count.",
+          "readOnly": true
+        },
+        "operatingSystem": {
+          "type": "string",
+          "description": "Operating System of the machine.",
+          "readOnly": true
+        },
+        "groups": {
+          "type": "array",
+          "description": "List of references to the groups that the machine is member of.",
+          "readOnly": true,
+          "items": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Reference to a group that this machine is member of. /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Migrate/projects/{projectName}/groups/{groupName}"
+          }
+        },
+        "createdTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Time when this machine was created. Date-Time represented in ISO-8601 format.",
+          "readOnly": true
+        },
+        "updatedTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Time when this machine was last updated. Date-Time represented in ISO-8601 format.",
+          "readOnly": true
+        },
+        "discoveredTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Time when this machine was discovered by Azure Migrate agent. Date-Time represented in ISO-8601 format.",
+          "readOnly": true
+        },
+        "disks": {
```

Comment:
>disks [](start = 9, length = 5)

```json
"disks": {
  "type": "object",
  "description": "Dictionary of disks attached to the machine. Key is ID of disk. Value is a disk object",
  "readOnly": true,
  "additionalProperties": {
    "$ref": "#/definitions/Disk"
  }
}
```
This is enough. I don't believe that extra `disk` level is need. Same for the network adapters dictionary.

Reply:
This causes the example validation to fail for Machines_Get.

Reply:
Err, no. The error has to do with incorrect examples. Please look closer.

Reply:
Fixed